### DISCUSSION TITLE
Can you use Semantic Kernel, in asp.net web api?

### DISCUSSION THREAD

**Discussion by fredprodibi** [microsoft/semantic-kernel#1716] [Q&A] [answered]
Hi, I m wondering if it's possible to use semantic kernel in an existing asp.net web api in .NET 7?

If yes do you know some documentation links, or tutorials or any help to get me started?

Thanks

**Comment by craigomatic** [comment, marked as answer]
Absolutely!

We have a sample that you can take a look at over here (the webapi folder is using aspnet): https://github.com/microsoft/semantic-kernel/tree/main/samples/apps/copilot-chat-app

Generally speaking, if you are using dotnet, SK can go anywhere that supports netstandard 2.0 or newer.

**Reply by PaulaScholz** [reply]
The webapp is not an asp.net core webapp.  It is React.

**Reply by craigomatic** [reply]
Correct, look at the webapi folder for asp.net (there are no razor pages, just api controllers)

**Reply by sambathrajv** [reply]
@craigomatic , it looks like the folder got moved somewhere. Can we get the latest path?

**Reply by sambathrajv** [reply]
Found the path. If anyone is looking for it, please use the project https://github.com/microsoft/chat-copilot.
